api/src/utilities/cron-job-starter.ts

```diff
@@ -35,27 +35,33 @@ export const startCronJob = (
   const repeatCron = cronString;
   const randomSecond = Math.floor(Math.random() * 30);
   const newCron = `${randomSecond * 2} ${repeatCron}`;
-  const job = new CronJob(newCron, () => {
-    void (async () => {
-      const currentCronJob = await prisma.cronJob.findFirst({
-        where: {
-          name: cronName,
-        },
-      });
-      // To prevent multiple overlapped jobs only run if one hasn't started in the last 5 minutes
-      if (
-        !currentCronJob ||
-        currentCronJob.lastRunDate <
-          dayjs(new Date()).subtract(5, 'minutes').toDate()
-      ) {
-        try {
-          await functionToCall();
-        } catch (e) {
-          logger.error(`${cronName} failed to run. ${e}`);
+  const job = new CronJob(
```

Comment:
I don't remember if this was part of this ticket or not. But are you planning on creating the cron job service as part of this?

Reply:
I am going to make a separate ticket for this work. Wouldn't be able to complete the cron job service work and testing before I am OOO